docusaurus.config.js

```diff
@@ -82,7 +82,8 @@ const config = {
         setupNote: `// ApiTokenIssuer, ApiAudience, ClientId and ClientSecret are optional.\n`,
       },
       go: {
-        importStatement: `. "github.com/openfga/go-sdk/client"`,
+        importStatement: `. "github.com/openfga/go-sdk"
```

Comment:
Added this to ensure that all snippets work fully when copied. This arguably could be `openfga "github.com/openfga/go-sdk"` but I went with this just to avoid any difference when writing `var body WriteAuthorizationModelRequest` vs say `var body ClientReadChangesRequest` as the former would then become `var body openfga.WriteAuthorizationModelRequest`

Reply:
Yeah `"github.com/openfga/go-sdk/client"` is still imported (line 86) just this is now first for formatting reasons